Content.Shared/Security/SecurityStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Content.Shared.Security
+{
+    /// <summary>
+    /// Status using in Criminal Records Console
```

Comment:
using -> used.

Please also comment on how new statuses require updates in methods such as `OnButtonPressed`